docs/logs/discovery/tailing-containers.md

```diff
@@ -0,0 +1,18 @@
+title: Shipping Container Logs
+description: Shipping Container Log Sources
+
+To enable automatic container logs shipping, start by clicking on the `Set Up` button. This will open the following flyout panel that allows configuring container log shipping:
+
+![Enable Container Logs](images/enable-container-logs.png)
+
+1. Trigger the "Automatically ship <group-name> logs" toggle
+2. Select the destination Logs App. If the Logs App doesn't exist, you'll be able to create one from the same screen
+3. Click the `Save Changes` button
+
+Log shipping is configured by the specific container image. This implies that any freshly created containers that match this image name will automatically have their logs shipped to Sematext Cloud.
+
+## Routing logs by container labels
+
+If you want to restrict the scope of log collection, it is possible to route the container logs by labels. You can either select a label from the list of already defined labels in this container or write a custom label. Labels can contain a key/value pair, or just a key.
```

Comment:
I assume that by "restricting scope" we mean "filter"?  Maybe that is a more direct way of saying it.
Actually, why are we calling this "Routing logs"?  Is this about restricting/filtering out containers whose logs we are not interested in, or is about sending some logs to one destination, and some to another?

Reply:
This is a good point. Will rework.

Reply:
@otisg Would `Filter by container labels` sound appropriate? I'm asking because Davor will have to adjust the UI accordingly.

Reply:
Yeah!  But what I see is also confusing is the ability to filter by container labels mixed with info about writing custom labels.  What are these custom labels attached to?  Not quite clear to me the reader how this works.

Reply:
> What are these custom labels attached to? 

I admit this might be confusing. We offer the ability to write custom labels for including logs of container that might not exist yet but could appear at some point in time in the future.

I'm not able to figure out what is the best way to convey this to users.